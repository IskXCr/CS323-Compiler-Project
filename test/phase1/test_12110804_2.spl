int _(){}

int /*;

int main()
{
    int a[3;
    int b[7];
    int c8];
}