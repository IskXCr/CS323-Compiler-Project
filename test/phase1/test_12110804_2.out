test/test_12110804_2.spl:5:11: [31merror:[0m missing closing bracket ']' [[31mB[0m]
test/test_12110804_2.spl:7:11: [31merror:[0m missing semicolon ';' [[31mB[0m]
test/test_12110804_2.spl:3:5: [31merror:[0m invalid function body [[31mB[0m]
       5 |    int a[[31m3[0m;
         |          [31m^[0m
       7 |    int c8[31m][0m;
         |          [31m^[0m
       3 |int [31mmain()[0m
         |    [31m^~~~~~[0m
